### DISCUSSION TITLE
Is there a way to ignore the DefaultExcludedFiles?

### DISCUSSION THREAD

**Discussion by Lenni009** [JamesIves/github-pages-deploy-action#1622] [Q&A]
I'm building a pipeline to allow gh-pages deployments from PR branches, using the PR's number.
So my gh-pages branch looks something like this:

```
| index.html
| .nojekyll
└ pr
  ├─1
  |  index.html
  |  .nojekyll
  └─2
     index.html
     .nojekyll
```

Since the PR numbers will always increase, and the `.nojekyll` files aren't cleaned up, I end up with a bunch of folders that only contain a `.nojekyll` file.

Is it somehow possible to circumvent this behaviour and clean up these files as well? Like the opposite of `clean-exclude`, `clean-include`, that could be part of the `with` property as well and could be set to the `DefaultExcludedFiles` by default.
Here's an example repo with my issue: https://github.com/Lenni009/action-test/tree/cd2616e

Thanks!

**Comment by JamesIves** [comment]
Today there's no way to do this, but there has been a proposal for this for some time: https://github.com/JamesIves/github-pages-deploy-action/blob/dev/src/git.ts#L184 

You could also consider simply removing the `.nojekyll` file prior to making your deployment as a workaround.

**Reply by christian-wolf-eks** [reply]
I highly suspect that jeust a single `/` is missing.
The problem is that the `rsync [...] --exclude .nojekyll` will recursively ignore all `.nojekyll` files, also in the subfolders. By making it `--exclude /.nojekyll`, it will not recusively descend.

> You could also consider simply removing the .nojekyll file prior to making your deployment as a workaround.

What do you mean by this? Could you please elaborate?
